README.rst

```diff
@@ -131,8 +134,7 @@ Viewing API Documentation without Sphinx
 
 Use the help function in the python interactive shell:
 
-.. code-block:: python
+.. code-block:: pycon
```

Comment:
Typo?

Reply:
No, this is the "console" type, since it has the `>>>` marks.